content/python/concepts/built-in-functions/terms/help/help.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'help()'
+Description: 'Display the documentation of the object passed as argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `help()` function accepts an object as a parameter, which is a module, function, class or keyword. After being called, the documentation of the requested object is displayed.
+If no argument is passed the interactive help utility starts up on the console.
+
+## Syntax
+
+The object is passed to the `help()` function as parameter:
+
+```pseudo
+help(object)
+```
+
+To start the interactive help utility, the function is called without argument:
+
+```pseudo
+help()
+```
+
+## Example
+
+Below is an example where a build-in function is passed as argument:
+
+```py
```

Comment:
It would be better if the output was presented in a separate ```shell block rather than as comments in the code. Just preface it with a sentence: This example produces the following output: